src/main/java/core/basesyntax/serviceimpl/ReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
```

Comment:
According to the checklist, the project structure should have a `service` package for holding services like Writer, Reader, Parser, etc., and a `service.impl` package for holding implementations of these services. Ensure that `ReportServiceImpl` is placed in the correct package to follow the project structure guidelines.